src/main/java/io/appium/java_client/android/StartsActivity.java

```diff
@@ -26,93 +26,26 @@
     /**
      * This method should start arbitrary activity during a test. If the activity belongs to
      * another application, that application is started and the activity is opened.
+     * <p>
+     * Usage:
+     * </p>
+     * <pre>
+     *     {@code
+     *     Activity activity = new Activity();
+     *     activity.setAppPackage("com.foo");
+     *     activity.setAppActivity(".bar");
+     *     driver.startActivity(activity);
+     *     }
+     * </pre>
      *
-     * @param appPackage      The package containing the activity. [Required]
```

Comment:
@email2vimalraj I'm against this change as it is. Please mark these methods `Deprecated` and with the description to use new ones